src/overlays/actors/ovl_En_Skj/z_en_skj.h

```diff
@@ -6,11 +6,97 @@
 
 struct EnSkj;
 
+typedef void (*EnSkjActionFunc)(struct EnSkj*, GlobalContext*);
+
 typedef struct EnSkj {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1B4];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s jointTable[19];
+    /* 0x0202 */ Vec3s morphTable[19];
+    /* 0x0274 */ EnSkjActionFunc actionFunc;
+    /* 0x0278 */ ColliderCylinder collider;
+    /* 0x02C4 */ u16 textId;
+    /* 0x02C6 */ s16 timer;
+    /* 0x02C8 */ s16 unk_2C8;
+    /*
+     * Sarias song skull kid as child always incriments
+     * Hostile skull kid as adult leaves the fight when it reaches zero
+     * Ocarina game fails the game when reaching zero
+     */
+    /* 0x02CA */ s16 multiuseTimer;
```

Comment:
I personally prefer to have simple comments that dont break up the struct. if theres anyway to condense this